test/controllers/sign.out.controller.spec.ts

```diff
@@ -158,6 +159,31 @@ describe("Sign Out controller", () => {
     });
   });
 
+  describe("GET with url params tests", () => {
```

Comment:
Was expecting the same tests that are currently in the "GET tests" test suite to be duplicated in this suite as per other PRs. Looks like there's only a subset here.

Reply:
The excluded tests are to do with whether the FEATURE_FLAG_ENABLE_SAVE_AND_RESUME_17102022 is active or not, are we really concerned with this for the parameterised tests?

Reply:
I agree, we're not concerned with the 'Save and Resume' feature flag tests. However these other tests are missing:

`renders the ${config.SIGN_OUT_PAGE} page, with ${config.SOLD_LAND_FILTER_PAGE} as back link`
`renders the ${config.SIGN_OUT_PAGE} page and correctly sets the previousPage field value`
`renders the ${config.SIGN_OUT_PAGE} page and does set a potentially malicious previous page URL`,

Reply:
I thought plan was to duplicate ALL the tests, so that when we come to delete the feature flag and related code, we just need to remove the entire block of the non params tests.

Reply:
Added these tests.

Reply:
To be discussed.

Reply:
Updated the tests to include all tests in both GET sections.